homeassistant/components/fibaro/__init__.py

```diff
@@ -65,7 +66,7 @@
 
 FIBARO_ID_LIST_SCHEMA = vol.Schema([cv.string])
 
-CONFIG_SCHEMA = vol.Schema({
+GATEWAY_CONFIG = vol.Schema({
     DOMAIN: vol.Schema({
```

Comment:
Why keep DOMAIN in GATEWAY_CONFIG ?

Reply:
I've removed it and it seems to work just as well ;-)